stubs/ldap3/ldap3/protocol/rfc4511.pyi

```diff
@@ -1,32 +1,26 @@
-from typing import Any as _Any
-
-# Enable when pyasn1 gets stubs:
-# from pyasn1.type.univ import Boolean, Choice, Enumerated, Integer, Null, OctetString, Sequence, SequenceOf, SetOf
-Boolean = _Any
-Choice = _Any
-Enumerated = _Any
-Integer = _Any
-Null = _Any
-OctetString = _Any
-Sequence = _Any
-SequenceOf = _Any
-SetOf = _Any
-
-LDAP_MAX_INT: int
-MAXINT: _Any
-rangeInt0ToMaxConstraint: _Any
-rangeInt1To127Constraint: _Any
-size1ToMaxConstraint: _Any
-responseValueConstraint: _Any
-numericOIDConstraint: _Any
-distinguishedNameConstraint: _Any
-nameComponentConstraint: _Any
-attributeDescriptionConstraint: _Any
-uriConstraint: _Any
-attributeSelectorConstraint: _Any
+from typing_extensions import Final as _Final
+
+from pyasn1.type.constraint import ConstraintsIntersection, SingleValueConstraint, ValueRangeConstraint, ValueSizeConstraint
+from pyasn1.type.namedtype import NamedTypes
+from pyasn1.type.namedval import NamedValues
+from pyasn1.type.tag import TagSet
+from pyasn1.type.univ import Boolean, Choice, Enumerated, Integer, Null, OctetString, Sequence, SequenceOf, SetOf
+
+LDAP_MAX_INT: _Final = 2147483647  # noqa: Y015  # name conflict
```

Comment:
flake8-pyi not happy about the alias. (`Final` is fine)

Reply:
Oh interesting. Yeah, flake8-pyi currently has no understanding of the fact that some imports might be aliased. I've been wondering about fixing it, but this kind of thing actually comes up surprisingly rarely in typeshed, and would involve adding some complexity to flake8-pyi.

Still, the fact that we have a legit false-positive here is a pretty good reason to fix it! 😀

Reply:
I've opened https://github.com/PyCQA/flake8-pyi/issues/316.